backend/dal/Helpers/Extensions/ProjectExtensions.cs

```diff
@@ -592,5 +586,27 @@ public static void Merge(this Entity.Project originalProject, Entity.Project upd
                 originalProject.UpdateProjectFinancials();
             }
         }
+
+        /// <summary>
+        /// Get a collection of responses that have changed from the original to the updated project.
+        /// </summary>
+        /// <param name="originalProject"></param>
+        /// <param name="updatedProject"></param>
+        /// <returns></returns>
+        public static IEnumerable<ProjectAgencyResponse> GetResponseChanges(this Entity.Project originalProject, Entity.Project updatedProject)
+        {
+            if (updatedProject == null) throw new ArgumentNullException(nameof(updatedProject));
+
+            var responses = new List<ProjectAgencyResponse>(updatedProject.Responses.Count());
+            foreach (var response in updatedProject.Responses)
+            {
+                var originalResponse = originalProject.Responses.FirstOrDefault(r => r.ProjectId == response.ProjectId && r.AgencyId == response.AgencyId);
+                if (originalResponse == null || originalResponse.Response != response.Response)
```

Comment:
is originalResponse.Response != response.Response going to compare the object references or the objects themselves?

Reply:
That should be to compare the response enum value.